packages/commonwealth/client/scripts/App.tsx

```diff
@@ -14,6 +14,7 @@ import { openFeatureProvider } from './helpers/feature-flags';
 import useAppStatus from './hooks/useAppStatus';
 import { trpc, trpcClient } from './utils/trpcClient';
 import { AddToHomeScreenPrompt } from './views/components/AddToHomeScreenPrompt';
+import Mava from './views/components/Mava/Mava';
```

Comment:
can you create index file for this component?

Reply:
Yeah... Sorry. Actually I meant to do that ...